Gemfile.lock

```diff
@@ -44,7 +44,7 @@ GEM
     rspec-support (3.10.0)
     sqlite3 (1.4.2)
     thread_safe (0.3.6)
-    tzinfo (1.2.8)
+    tzinfo (1.2.9)
```

Comment:
Did you need the latest version for this change or are you just updating this to stay current?

Reply:
Just staying current. It's not an intentional change.